<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstRunWizard</class>
 <widget class="QWizard" name="FirstRunWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="welcomePage">
   <property name="title">
    <string>Welcome!</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Welcome to the MXit PC Client first-run wizard.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>This wizard will guide you through the various things needed to set up your client such as themes, logs, gateways, other general options and registration (if you are not registered already)</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>If you wish to skip this step, click cancel, otherwise click next to continue</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="themePage">
   <property name="title">
    <string>Theme Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Your MXit Client can be personalised with themes, choose the directory where your themes are stored</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="conversationsPage">
   <property name="title">
    <string>Conversations Log Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Logs of all your conversations can be saved to your computer. Leave blank if you don't wish to log </string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="generalPage">
   <property name="title">
    <string>General Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>There are a few general settings one can change to make your MXit PC client look and feel the way you want it to</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <property name="title">
    <string>Registration</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Thank you for completeing the first-run wizard. If you are not already registed on MXit, you can do so here.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Otherwise click Finish to continue to the login dialog. Enjoy :)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
